src/sentry/models/project.py

```diff
@@ -245,6 +245,7 @@ class Project(Model, PendingDeletionMixin, OptionMixin, SnowflakeIdMixin):
     # projects that were created before this field was present
     # will have their first_event field set to date_added
     first_event = models.DateTimeField(null=True)
+    project_template = FlexibleForeignKey("sentry.ProjectTemplate", null=True)
```

Comment:
Someone like Dan would know better than me but idk if you can add this _and_ create the new table in one go

Reply:
yeah, i wasn't sure if i should do all 3 model changes in 1 pr or in 3 PRs. I can ping dan and see if this makes sense.

fwiw the migration did work locally.. unclear if that means it'll work in prod or not though 😅

Reply:
ah ha! it seems that django uses the `operations` key to decide the order it applies the migrations: https://github.com/getsentry/sentry/pull/69816/files#diff-705160f3302cfba81a9ad4b3182878d45a36c8a5f3f55d5be85d5ebde083e309R32 - since django automates all this they figured out the application order for me.

Reply:
This is fine, you can add an FK and the table in the same migration